### PR TITLE
Update BMLacq812.xml

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
Updated Contents and PhysDesc according to Marrassinis catalogue. I could not check, because the images have not been properly displayed to me on the screen.

### CODE REVIEW

File: FlorenceBML/BMLacq812/BMLacq812.xml [hunk 1]
```diff
@@ -72,6 +244,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
         <revisionDesc>
```
Comment (eu-genia) [inline FlorenceBML/BMLacq812/BMLacq812.xml:244]: add Geez to langusage

File: FlorenceBML/BMLacq812/BMLacq812.xml [hunk 2]
```diff
@@ -37,6 +38,177 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                                <msItem xml:id="ms_i1.2.7">
+                                    <locus from="148r" to="148v"/>
+                                    <title ref="LIT2274salota"/>
+                                </msItem>
+                                <msItem xml:id="ms_i1.2.8">
+                                    <locus from="148v" to="150r"/>
+                                    <title ref="LIT2250salota"/>
+                                </msItem>
+                                <msItem xml:id="ms_i1.2.9">
+                                    <locus from="150" to="151r"/>
```
Comment (eu-genia) [inline FlorenceBML/BMLacq812/BMLacq812.xml:88]: 150r?

**Review by eu-genia — CHANGES_REQUESTED** [review]
very nice, just small things; images should be visible now, too

File: FlorenceBML/BMLacq812/BMLacq812.xml (hunk 1 above)
Comment (CarstenHoffmannMarburg) [reply FlorenceBML/BMLacq812/BMLacq812.xml:244]: Done.

Comment (CarstenHoffmannMarburg) [reply FlorenceBML/BMLacq812/BMLacq812.xml:244]: Done.

Comment (CarstenHoffmannMarburg) [reply FlorenceBML/BMLacq812/BMLacq812.xml:244]: But I cannot see the full images but rather the small item on the bottom line only. It does not work properly in neither in Chrome nor in firefox. Maybe I should shut down the computer and try a new start.

File: FlorenceBML/BMLacq812/BMLacq812.xml (hunk 2 above)
Comment (CarstenHoffmannMarburg) [reply FlorenceBML/BMLacq812/BMLacq812.xml:88]: I took it, as it is given in Marrassinis catalogue. If the images are available to me, I will update the folio numbering.

File: FlorenceBML/BMLacq812/BMLacq812.xml (hunk 1 above)
Comment (eu-genia) [reply FlorenceBML/BMLacq812/BMLacq812.xml:244]: I see it now too, it is very strange, as other manuscripts seem to work, I will try to explore when I am in office what the bug is with this MS

**Comment by eu-genia** [comment]
Geez to be added to langUsage

I am still not sure what is wrong with the viewer for this MS, I have opened an issue and will try to find out but so far am clueless

The images can be seen at the external drive \aai-hlz-research\MANUSCRIPTS\BML_007 
(or also one by one online 
https://betamasaheft.eu/iiif/Laurenziana/BML_007/BML_007-001.tif/full/full/0/default.jpg
https://betamasaheft.eu/iiif/Laurenziana/BML_007/BML_007-002.tif/full/full/0/default.jpg
etc)

**Comment by eu-genia** [comment]
Please change the `idno `to point to `facs="Laurenziana/812/BML_007-"` (I had to change the folder, there is some bug in the old one)

**Review by eu-genia — APPROVED** [review]

File: FlorenceBML/BMLacq812/BMLacq812.xml (hunk 2 above)
Comment (CarstenHoffmannMarburg) [reply FlorenceBML/BMLacq812/BMLacq812.xml]: I saw the images and committed some changes.

**Pull request merged**